### PR TITLE
Updated stig::system_auth to manage /etc/pam.d/postlogin-ac

### PR DESCRIPTION
**Pull request by kdshah1**

### CODE REVIEW

File: attributes/default.rb [hunk 1]
```diff
@@ -1664,3 +1664,10 @@
   shutdown
   sync
 ]
+
+default['stig']['pam_d']['config']['postlogin_ac_rules'] = [
+  'session     [success=1 default=ignore] pam_succeed_if.so service !~ gdm* service !~ su* quiet',
+  'session     required      pam_lastlog.so showfailed'
+  # 'session     [default=1]   pam_lastlog.so nowtmp showfailed',
```
Comment (isuftin) [inline attributes/default.rb]: Remove commented out values here

Comment (isuftin) [reply attributes/default.rb]: If I'm reading this correctly, and this is coming from https://static.open-scap.org/ssg-guides/ssg-rhel7-guide-pci-dss.html#xccdf_org.ssgproject.content_rule_display_login_attempts

... those lines should not be commented?

**Comment by isuftin** [comment]
I don't believe this is valid for RHEL 6. This seems to only be valid for RHEL 7. You probably will want to do a major version check before writing this file.

**Comment by isuftin** [comment]
Ack. Accidentally closed.

**Comment by isuftin** [comment]
Please add ChefSpec testing for this @ https://github.com/USGS-CIDA/stig/blob/master/spec/system-auth_spec.rb

File: attributes/default.rb (hunk 1 above)
Comment (kdshah1) [reply attributes/default.rb]: Take a look at this: https://vaulted.io/library/disa-stigs-srgs/RHEL_7_STIG/V-72275

Comment (isuftin) [reply attributes/default.rb]: Right. Basically the same info as my link, but my link does provide the fix. Though your fix does differ from the one suggested in my link a bit.

**Comment by kdshah1** [comment]
added a check for RHEL and version >=7. Removed the commented line.

**Pull request merged**